ci/README.md

```diff
@@ -77,8 +77,8 @@ them to merely debug issues.
 
 In the previous section we've mentioned using different runners, now in this section we'll go through each type of runner used.
 
-- Cost free runners:  Those are the runners provided by GIthub itself, and
-  those are fairly small machines with no virtualization capabilities enabled - 
+- Cost free runners:  Those are the runners provided by Github itself, and
+  those are fairly small machines with no virtualization capabilities enabled. 
```

Comment:
Well, as you're already touching this, I think it's worth updating the documentation as the Cost free runners now support virtualization.

Reply:
@fidencio changes applied